examples/screenshare/src/helpers.js

```diff
@@ -20,9 +20,7 @@ function createScreenTrack(height, width) {
       height: height,
       width: width
     }
-  }).then(function(stream) {
-    return new Video.LocalVideoTrack(stream.getVideoTracks()[0]);
-  });
```

Comment:
You don't need to change this function in order to listen to `onended`. You can just do it like this:
```js
const screenTrack = await createScreenTrack(...);
screenTrack.mediaStreamTrack.onended = stopScreenSharing;
```

Reply:
Thanks Manjesh! It is done